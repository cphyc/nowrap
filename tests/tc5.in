123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas vitae imperdiet tellus. Vivamus sagittis mauris in
magna aliquet dapibus. Donec augue libero, commodo ac eleifend sed, ultricies quis lorem. Duis nisl arcu, facilisis a
cursus ac, adipiscing eu metus. Sed cursus enim et metus sodales ornare. Donec elementum ipsum eu augue luctus laoreet.
Duis nec neque eu metus venenatis vestibulum. Aenean arcu arcu, scelerisque fringilla suscipit vitae, mollis ac quam.
Nam eleifend rutrum tellus ac venenatis.

You should see the Greek word 'kosme':       "κόσμε"3456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|

This is a randomly selected 2-column Hirigana character: "き"23456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789| 

Some Thai combining text (appears to break Vim/Terminal.app?): กั56789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|
The above Thai char is actually two separate characters where one of them has zero-width

