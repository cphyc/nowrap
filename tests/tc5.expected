123456789|123456789|123456789|123456789|123456789|123456789|123456789|12

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas vitae 
magna aliquet dapibus. Donec augue libero, commodo ac eleifend sed, ultr
cursus ac, adipiscing eu metus. Sed cursus enim et metus sodales ornare.
Duis nec neque eu metus venenatis vestibulum. Aenean arcu arcu, sceleris
Nam eleifend rutrum tellus ac venenatis.

You should see the Greek word 'kosme':       "κόσμε"3456789|123456789|12

This is a randomly selected 2-column Hirigana character: "き"23456789|12

Some Thai combining text (appears to break Vim/Terminal.app?): กั56789|12
The above Thai char is actually two separate characters where one of the

